src/helpers/index.js

```diff
@@ -47,6 +50,29 @@ const formatRepValue = (value, callback = logError) => dispatch => callback(form
 
 const formatEthValue = (value, callback = logError) => dispatch => callback(formatEther(value))
 
+const getRepTokens = (callback = logError) => (dispatch) => {
+  dispatch(getRep((err) => {
+    if (err) return callback({ err })
+    return callback({ err: null })
+  }))
+}
+
+const placeTradeOnMarket = (marketId, outcomeId, callback = logError) => (dispatch) => { // untested
+  dispatch(placeTrade(marketId, outcomeId, false, true, (err) => {
+    if (err) return callback({ err })
+    return callback()
+  }))
+}
+
+const getMarketCosts = (callback = logError) => (dispatch) => {
+  const { universe } = store.getState()
+
+  dispatch(augur.createMarket.getMarketCreationCostBreakdown({ universe: universe.id }, (err, marketCreationCostBreakdown) => {
+    if (err) return callback({ err })
+    return callback({ err: null, data: marketCreationCostBreakdown })
+  }))
+}
```

Comment:
this isn't a dispatch thing, I don't think
just call 
augur.createMarket.getMarketCreationCostBreakdown({ universe: universe.id }, (err, marketCreationCostBreakdown) ....